source/Playnite.DesktopApp/ViewModels/DesktopAppViewModel.cs

```diff
@@ -43,6 +43,7 @@ public partial class DesktopAppViewModel : MainViewModelBase, IDisposable, IMain
         protected bool ignoreCloseActions = false;
         protected bool ignoreSelectionChanges = false;
         private readonly SynchronizationContext context;
+        private readonly Random random = new Random();

```

Comment:
It would probably make sense to use static `GlobalRandom` class here. Which then can be used in RandomGameSelectViewModel as well.

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3032/commits/3e3b39498df2812adb0187895b6972aa935a0ce3